Source/Preview/App/Fuse.Simulator/SimulatorClient.uno

```diff
@@ -113,7 +113,7 @@ namespace Fuse.Simulator
 		readonly Uno.Threading.ConcurrentQueue<IBinaryMessage> _messagesToClient = new Uno.Threading.ConcurrentQueue<IBinaryMessage>();
 		readonly Thread _readWorker;
 		readonly Thread _writeWorker;
-		readonly IDisposable _alsoReceieveMessagesFromPipe;
+		//readonly IDisposable _alsoReceieveMessagesFromPipe;
```

Comment:
Just delete the declaration, no point in keeping it commented out...

Reply:
Done - thanks!